css/css-positioning/css-positioning_absolute-no-advanced-requirements/css/styles.css

```diff
@@ -41,24 +41,43 @@ main {
 
 .box--top-center {
   background-color: #85c1e9;
+  position: absolute;
+  top: 0px;
+  right: 220px;
 }
 
 .box--top-right {
   background-color: #5dade2;
+  position: absolute;
+  right: 0;
+  top: 0;
 }
 
 .box--center {
   background-color: #3498db;
+  position: absolute;
+  right: 220px;
+  top: 220px;
+  bottom: 220px;
```

Comment:
here again, great job at figuring out the right numbers (wonder how you did it), for me easier would be the following: 
left: 50%;
   top: 50%;
   transform: translate(-50%, -50%);